api_docs/changelog.md

```diff
@@ -20,6 +20,11 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 8.0
 
+**Feature level 192**
+
+* [`GET /events`](/api/get-events): Stream creation events are now
+  sent when guest users gain access to a stream by being subscribed.
```

Comment:
Should this say "public stream"? Because they did get the event when they were subscribed to private streams, correct? So just to be extra clear ...

> [`GET /events`]: Stream creation events are now sent when guest users gain access to a public stream by being subscribed. Guest users previously only received these events when subscribed to private streams.

Reply:
Yes, it should say "public stream". Updated as per the suggestion.